resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,407 @@
+        "ec2:ModifyInstanceAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeLocalGateway",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeLocalGatewayRouteTables",
+        "ec2:DescribeLocalGatewayRouteTableVirtualInterfaceGroupAssociations",
+        "ec2:DescribeLocalGatewayRouteTableVpcAssociations",
+        "ec2:DescribeLocalGateways",
+        "ec2:DescribeLocalGatewayVirtualInterfaceGroups",
+        "ec2:DescribeLocalGatewayVirtualInterfaces"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeTransitGateway",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeTransitGatewayAttachments",
+        "ec2:DescribeTransitGatewayConnectPeers",
+        "ec2:DescribeTransitGatewayConnects",
+        "ec2:DescribeTransitGatewayMulticastDomains",
+        "ec2:DescribeTransitGatewayPeeringAttachments",
+        "ec2:DescribeTransitGatewayRouteTables",
+        "ec2:DescribeTransitGateways",
+        "ec2:DescribeTransitGatewayVpcAttachments",
+        "ec2:GetTransitGatewayAttachmentPropagations",
+        "ec2:GetTransitGatewayPrefixListReferences",
+        "ec2:GetTransitGatewayRouteTableAssociations",
+        "ec2:GetTransitGatewayRouteTablePropagations"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "ManageMulticastDomain",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:GetTransitGatewayMulticastDomainAssociations",
+        "ec2:SearchTransitGatewayMulticastGroups"
+      ],
+      "Resource": "arn:aws:ec2:*:*:transit-gateway-multicast-domain/*"
+    },
+    {
+      "Sid": "SearchTransitGatewayRoutes",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:SearchTransitGatewayRoutes"
+      ],
+      "Resource": "arn:aws:ec2:*:*:transit-gateway-route-table/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeVpc",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVpcClassicLink",
+        "ec2:DescribeVpcClassicLinkDnsSupport",
+        "ec2:DescribeVpcEndpointConnectionNotifications",
+        "ec2:DescribeVpcEndpointServiceConfigurations",
+        "ec2:DescribeVpcEndpointServices",
+        "ec2:DescribeVpcPeeringConnections"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeVpcAttribute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVpcAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:vpc/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeVpcEndpointServicePermission",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVpcEndpointServicePermissions"
```

Comment:
What is this used for?

Reply:
since we probably won;t need to see who's connected to their service, will remove this permissio